test/integration/chunking/test/index.test.js

```diff
@@ -37,12 +40,6 @@ describe('Chunking', () => {
     expect(existsChunkNamed('framework')).toBe(true)
   })
 
-  it('should create a library chunk for lodash', () => {
```

Comment:
Removing this test as we're no longer naming library chunks in a predictable way. I could use the generated hash, here, but I'm afraid that would make this test too fragile.